besu/src/main/java/org/hyperledger/besu/cli/config/NetworkName.java

```diff
@@ -77,6 +80,14 @@ public BigInteger getNetworkId() {
     return networkId;
   }
 
+  /**
+   * Sets network id.
+   */
+  public void setNetworkId(final BigInteger networkId) {
```

Comment:
Yes this is only for Ephemery Network. Will update the comment to reflect that.

Reply:
I think it would be better to have a programmatic check - maybe add the network name as a parameter to this method, and only modify the networkId if the name == EPHEMERY

otherwise throw an exception maybe? or just log a warning